.github/workflows/workflows.yml

```diff
@@ -0,0 +1,432 @@
+          #   with:
+          #     miniconda-version: "latest"
+          #     activate-environment: datalad
+          #     channels: conda-forge
+          #     channel-priority: flexible
+          #     allow-softlinks: true
+          #     show-channel-urls: true
+          #     use-only-tar-bz2: true
+
+          # - name: Check conda installation
+          #   shell: bash -l {0}
+          #   run: |
+          #     conda info
+          #     conda list
+
+      - name: Install dependencies
+        shell: bash -l {0}
+        run: |
+          sudo apt-get update -y && sudo apt-get upgrade -y
+          sudo apt-get install -y git-annex
+          python -m pip install --upgrade pip
+          pip install datalad==0.14.7
+
+      - name: WTF!?
+        run: |
+         datalad wtf
+
+      - name: "cached data"
+        uses: actions/cache@v2
+        id: cache
+        with:
+          path: /tmp/.cache/data
+          key: ds003-v4-${{ runner.os }}-${{ github.job }}
+          restore-keys: |
+            ds003-v4-${{ runner.os }}-
+            ds003-v4-
+            ds003-
+
+      - name: "Install fMRIPrep derivatives of ds000003 and reference data"
+        # if: steps.cache.outputs.cache-hit != 'true'
+        run: |
+          cd /tmp/.cache/data
+          datalad install -r -s https://gin.g-node.org/shashankbansal56/fitlins_tests
+          datalad update --merge -d fitlins_tests
+          cd fitlins_tests
+          datalad get ds003/fmriprep/sub-0{1,2,3}/func/*_space-MNI152NLin2009cAsym_desc-*.nii.gz \
+                                    ds003/fmriprep/sub-0{1,2,3}/func/*_desc-confounds_*.tsv \
+                                    ds003/fmriprep/dataset_description.json \
+                                    ds003/fmriprep/sub-*/*/*.json
+          datalad get -r ds003/nistats_smooth/ ds003/afni_smooth/ ds003/afni_blurto/ ds003/nistats_blurto/
+          datalad status
+
+      - name: "Check installed derivatives data"
+        run: |
+          cd /tmp/.cache/data/fitlins_tests/ds003
+          git log --oneline --graph
+          du -sh .
+
+  build_docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Check python version and upgrade pip
+        run: |
+          python --version
+          python -m pip install -U pip setuptools
+
+      - name: Install pandoc
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y pandoc
+
+      - uses: actions/checkout@v2
+      - name: Install fitlins
+        run: |
+          cd $GITHUB_WORKSPACE
+          pip install ".[docs]"
+
+      - name: Build docs
+        run: |
+          cd $GITHUB_WORKSPACE
+          make html
+          ls
+          pwd
+
+      - name: Store artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: docs
+          path: docs/_build/html
```

Comment:
Should we just use ReadTheDocs PR builds?

Reply:
What is that? Like a separate workflow that runs on readthedocs PRs?

Reply:
It's just something to enable in RTD. Unless there's a specific reason to have this in GitHub actions, I would move to that, as it's more predictive of whether we will be able to build the docs after a PR is merged. I'll go ahead and enable it and we can see.

Reply:
```suggestion
```